lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -33,12 +34,34 @@ def create_inference_job(
     return job_response
 
 
-@router.post("/evaluate/", status_code=status.HTTP_201_CREATED)
-def create_evaluation_job(
+@router.post("/annotate/", status_code=status.HTTP_201_CREATED)
+def create_annotation_job(
     service: JobServiceDep,
-    job_create_request: JobEvalCreate,
+    job_create_request: JobAnnotateCreate,
     request: Request,
-    response: Response
+    response: Response,
+) -> JobResponse:
+    # Lumigator's opinion on the best summarization model
+    # and the one that should generate annotations.
+    # In the future, we could expose this via a config file
```

Comment:
I'd leave out what we plan to do in the future as this could be confusing in the code, relegating planning to issues or RFCS, or instead might leave this as a TODO